client-app/shared/layout/components/search-bar/search-bar.vue

```diff
@@ -213,7 +213,7 @@ async function searchAndShowDropdownResults(): Promise<void> {
   hideSearchDropdown();
 
   if (
-    searchPhrase.value === "" ||
+    searchPhrase.value.trim() === "" ||
```

Comment:
May be we should optimize the code by storing the trimmed string in a variable trimmedSearchPhrase or smth, to avoid multiple calls of trim().

Reply:
I agree it's better to have single computed with trim instead

Reply:
I agree it's better to have single computed with single `trim` instead